lib/global-styles.php

```diff
@@ -304,3 +302,5 @@ function gutenberg_global_styles_include_support_for_wp_variables( $allow_css, $
 add_filter( 'force_filtered_html_on_import', 'gutenberg_global_styles_force_filtered_html_on_import_filter', 999 );
 add_filter( 'safecss_filter_attr_allow_css', 'gutenberg_global_styles_include_support_for_wp_variables', 10, 2 );
 // This filter needs to be executed last.
+
+add_filter( 'enqueue_block_editor_assets', 'gutenberg_load_css_custom_properties' );
```

Comment:
I think this was of loading styles is kind of deprecated to the iframing. cc @ellatrix

Reply:
For them to load in the iframe, the styles need to be enqueued here:

https://github.com/WordPress/gutenberg/blob/e5234418780871ccf848bdbf21d8dd9a67b6963b/lib/client-assets.php#L710-L780

Reply:
Hey @ellatrix I've pushed the changes to use the function you mention, but it looks like it enqueues the given styles within the iframe. Or perhaps I need to do more things?

In this PR I'm actually trying to do the opposite: enqueue the styles outside the iframe, so they're available for the global styles sidebar.

This is the setup:

- Styles for the content, within the iframe: CSS vars + block styles. This is working fine and it's done at `gutenberg_experimental_global_styles_settings` (adds them to editor settings' styles key).
- Styles for the sidebar: only the CSS vars. So far, we've been using the same method than the content's styles but with an additional flag (`__experimentalNoWrapper`) so they weren't wrapped by `.editor-styles-wrapper` and so targeted the sidebar. This is what I'm struggling to see how to do in an iframed editor.

Reply:
Talked with ella in private and can confirm that using `enqueue_block_editor_assets` is fine. So, switching this back to the original direction.

Reply:
`enqueue_block_editor_assets` is for the editor UI only, not the iframe. If you need it to load in both, you'll need to use both.